dali/operators/decoder/inflate/inflate.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "dali/operators/decoder/inflate/inflate.h"
+
+namespace dali {
+
+DALI_SCHEMA(experimental__Inflate)
+    .DocStr(R"code(Inflates/decompresses the input using specified decompression algorithm.
+
+The input must be a 1D tensor of bytes (uint8). Passing the ``shape`` and ``dtype`` of the
+decompressed samples is required.
+
+Each input sample can either be a single compressed chunk or consist of multiple
+compressed chunks that have the same shape and type when inflated, so that they can be
+be merged into a single tensor where the leftmost extent of the tensor corresponds
```

Comment:
```suggestion
be merged into a single tensor where the outermost extent of the tensor corresponds
```

Reply:
The real use case it targets are 3D sequences of single channel HW images. Each image is compressed separately and the results are concatenated. Using `chunks_offsets` or `chunks_sizes` you can do the reverse: uncompress the chunks into a sequence.

Reply:
`outermost` still applies - we never use `leftmost` in this context (the word appears 6 times in our code base and refers to position within image/array, not to dimensions). There are 67 occurrences of `outermost`.

Reply:
I've resolved that disccusion by misclick and couldn't find this anywhere. :p
Anyway, renamed it.